internal/actor/http.go

```diff
@@ -102,7 +102,7 @@ func (t *HTTPTransport) RoundTrip(req *http.Request) (*http.Response, error) {
 // 🚨 SECURITY: This should *never* be called to wrap externally accessible handlers (i.e.
 // only use for internal endpoints), because internal requests can bypass repository
 // permissions checks.
-func HTTPMiddleware(next http.Handler) http.Handler {
+func HTTPMiddleware(logger log.Logger, next http.Handler) http.Handler {
 	return http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
```

Comment:
We might want to sub-scope this:

```suggestion
func HTTPMiddleware(logger log.Logger, next http.Handler) http.Handler {
    logger = logger.Scoped("actors", "middleware for parsing actor from HTTP headers")

	return http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
```

Reply:
TIL